internal/controllers/user_devices_controller.go

```diff
@@ -1668,7 +1917,8 @@ func (udc *UserDevicesController) UpdateNFTImage(c *fiber.Ctx) error {
 type VehicleMintRequest struct {
 	NFTImageData
 	// Signature is the hex encoding of the EIP-712 signature result.
-	Signature string `json:"signature" validate:"required"`
+	Signature string               `json:"signature" validate:"required"`
+	SACDInput *contracts.SacdInput `json:"sacd_input,omitempty"`
```

Comment:
maybe use separate  struct so we can set JSON tags

Reply:
```
+  github_com_DIMO-Network_devices-api_internal_contracts.SacdInput:
+    properties:
+      expiration:
+        $ref: '#/definitions/big.Int'
+      grantee:
+        items:
+          type: integer
+        type: array
+      permissions:
+        $ref: '#/definitions/big.Int'
+      source:
+        type: string
+    type: object
```
The names actually turned out okay! But the types are kinda fried.

It might be a worthwhile follow-up to do a little validation on these guys.